impl/src/main/java/org/jboss/arquillian/container/weld/embedded/mock/FlatDeployment.java

```diff
@@ -68,6 +71,12 @@ protected void configureServices(Environment environment) {
             services.add(EjbServices.class, new MockEjBServices());
             services.add(SecurityServices.class, new MockSecurityServices());
         }
+        if (environment.equals(Environments.SE)) {
+            ExternalConfigurationBuilder configurationBuilder = new ExternalConfigurationBuilder()
+                // Use relaxed construction by default
+                .add(ConfigurationKey.RELAXED_CONSTRUCTION.get(), true);
```

Comment:
These are just common system properties so you can pass them via surefire-plugin.